packages/teleport/src/Discover/Shared/Timeout.tsx

```diff
@@ -1,37 +1,35 @@
 import React, { useEffect, useState } from 'react';
-import styled from 'styled-components';
 
 interface TimeoutProps {
   timeout: number; // ms
+  message?: string;
 }
 
-const Container = styled.div`
-  margin-top: 20px;
-`;
-
-export function Timeout(props: TimeoutProps) {
+export function Timeout({ timeout, message }: TimeoutProps) {
   const [, setCount] = useState(0);
 
   useEffect(() => {
     const interval = window.setInterval(() => {
-      if (Date.now() >= props.timeout) {
+      if (Date.now() >= timeout) {
         clearInterval(interval);
       }
 
       setCount(count => count + 1);
     }, 1000);
 
     return () => clearInterval(interval);
-  }, [props.timeout]);
+  }, [timeout]);
 
   const { minutes, seconds } = millisecondsToMinutesSeconds(
-    props.timeout - Date.now()
+    timeout - Date.now()
   );
 
   return (
-    <Container>
-      This script is valid for another {minutes}:{seconds}.
-    </Container>
+    <span>
+      {message
+        ? `${message} ${minutes}:${seconds}`
+        : `This script is valid for another ${minutes}:${seconds}`}
+    </span>
```

Comment:
What do you think about adding
```typescript
message = message || 'This script is valid for another';
```

and making this 

```suggestion
    <span>
      {message} {minutes}:{seconds}
    </span>
```

Reply:
much better, thanks